dotCMS/src/main/java/com/dotmarketing/util/TaskLocatorUtil.java

```diff
@@ -280,9 +281,22 @@ public static void removeFixTask(Class<?> clazz){
 		ret.add(Task05220MakeFileAssetContentTypeBinaryFieldIndexedListed.class);
         ret.add(Task05225RemoveLoadRecordsToIndex.class);
         ret.add(Task05300UpdateIndexNameLength.class);
-		return ret;
+        
+        
+        return ret.stream().sorted(classNameComparator).collect(Collectors.toList());
+
+
 	}
 
+	
+    final static private Comparator<Class<?>> classNameComparator = new Comparator<Class<?>>() {
+        public int compare(Class<?> o1, Class<?> o2) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'o1' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5632525)